dali/datest/datest.cpp

```diff
@@ -3309,7 +3309,7 @@ void testMultiConnect()
 }
 
 #if 0
-LogMsgCategory const daliAuditLogCat(MSGAUD_audit, MSGCLS_information, DefaultDetail);
+LogMsgCategory const MCauditInfo(MSGAUD_audit, MSGCLS_information, DefaultDetail);
```

Comment:
@shamser - this shouldn't be needed, it's already declared / will be same as one in jlog.